src/dir.targets

```diff
@@ -157,9 +154,39 @@
     <DebugEngines>{2E36F1D4-B23C-435D-AB41-18E608940038}</DebugEngines>
   </PropertyGroup>
 
+  <Target Name="CheckTestCategories"
+          AfterTargets="Test"
+          Condition="'$(IsTestProject)'=='True' And '$(SkipTests)'!='True'">
+    <PropertyGroup>
+      <!-- By default project has inner- and outer- loop tests. -->
+      <TestCategories Condition="'$(TestCategories)'==''">InnerLoop;OuterLoop</TestCategories>
```

Comment:
Perhaps we should just default this to InnerLoop given that InnerLoop will be included when we do an OuterLoop test run.